app/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Application entry-point"""
+
+from flask import Flask
+from flask_bcrypt import Bcrypt
+from flask_sqlalchemy import SQLAlchemy
+from app.authentication import authentication_app
+from config import ENVIRONMENT_OBJECT
+
+APP = Flask(__name__)
+APP.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
```

Comment:
Why not make **SQLALCHEMY_TRACK_MODIFICATIONS** part of your config

Reply:
Ooh that makes sense..will fix it